web/handlers.go

```diff
@@ -235,10 +235,7 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Strict-Transport-Security", fmt.Sprintf("max-age=%d", *c.App.Config().ServiceSettings.TLSStrictTransportMaxAge))
 	}
 
-	cloudCSP := ""
-	if c.App.Channels().License().IsCloud() {
-		cloudCSP = " js.stripe.com/v3"
-	}
+	cloudCSP := " js.stripe.com/v3"
```

Comment:
This wasn't always guarded by cloud and a few concerned customers brought it up. Wonder if we should have a global config (other than a FF) that can disable this whole feature?

Reply:
Good point, also related to my comment above about feature flags - this would be a good reason to change from feature flag to a config setting